client/src/components/atoms/Dropdown.tsx

```diff
@@ -0,0 +1,39 @@
+import './style/Dropdown.scss';
```

Comment:
Don't need to import `Dropdown.scss` since `components/atoms/style/index.scss` already imports `Dropdown.scss`